### PR TITLE
[SPARK-5654] Integrate SparkR into Apache Spark

### PR DESCRIPTION
**Pull request by davies**
This PR integrate SparkR-pkg [1] in Spark, it include almost complete API for Spark core and SparkSQL.

The following things could be done later (out of this PR):
1. generating documentation
2. R lint
3. API audit
4. more tests

Known issues:
1. does not work in YARN cluster mode

[1] github.com/amplab-extras/SparkR-pkg

### CODE REVIEW

File: R/pkg/DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Package: SparkR
+Type: Package
+Title: R frontend for Spark
+Version: 0.1
+Date: 2013-09-09
+Author: Shivaram Venkataraman
+Maintainer: Shivaram Venkataraman <shivaram@cs.berkeley.edu>
```
Comment (davies) [inline R/pkg/DESCRIPTION]: @shivaram Should we update these fields?

File: dev/run-tests [hunk 2]
```diff
@@ -185,48 +185,58 @@ echo "========================================================================="
 
 CURRENT_BLOCK=$BLOCK_SPARK_UNIT_TESTS
 
-{
-  # If the Spark SQL tests are enabled, run the tests with the Hive profiles enabled.
-  # This must be a single argument, as it is.
-  if [ -n "$_RUN_SQL_TESTS" ]; then
-    SBT_MAVEN_PROFILES_ARGS="$SBT_MAVEN_PROFILES_ARGS -Phive -Phive-thriftserver"
-  fi
+#{
```
Comment (davies) [inline dev/run-tests]: Rollback before merge

**Comment by davies** [comment]
@shivaram @pwendell @rxin I think this PR is ready to review.

**Comment by shivaram** [comment]
Thanks @davies  Lets close this PR and I will open a fresh one -- I want to give a better description to the PR and acknowledge all the contributors etc.

**Comment by concretevitamin** [comment]
@shivaram What's the timeline we are looking at? 

If possible I'd like to take a close look at this next week / after next week.

**Comment by shivaram** [comment]
@concretevitamin We still have a few PRs open at SparkR-pkg and once they are merged I'll open a fresh PR here. It'll probably be end of this week to get there and after that we'll keep it open here to give enough time for code review.

**Comment by davies** [comment]
Close this one, will open an new one by @shivaram

**Pull request closed without merging**